opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go

```diff
@@ -128,15 +126,15 @@ func resourceCTSTrackerRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	trackers, err := tracker.List(ctsClient, listOpts)
 	if err != nil {
```

Comment:
how  about put two conditions together?

Reply:
You mean, check for both 404 and empty trackers?

Reply:
after reviewing the [golangsdk](https://github.com/huaweicloud/golangsdk/blob/master/openstack/cts/v1/tracker/requests.go#L21-L33), the 404 error happens rarely and your patch also includes the error, so feel free about my comment above.

Reply:
Just from my point of view, 404 for the tracker list should be impossible. So you're okay with merging PR as-is?

Reply:
sure